target_csv/tests/test_core.py

```diff
@@ -1,25 +0,0 @@
-"""Tests standard target features using the built-in SDK tests library."""
-
-from typing import Any, Dict
-
-from singer_sdk.testing import get_standard_target_tests
-
-from target_csv.target import TargetCSV
-
-SAMPLE_CONFIG: Dict[str, Any] = {
-    # TODO: Initialize minimal target config
-}
-
-
-# Run standard built-in target tests from the SDK:
-def test_standard_target_tests():
-    """Run standard target tests from the SDK."""
-    tests = get_standard_target_tests(
-        TargetCSV,
-        config=SAMPLE_CONFIG,
-    )
```

Comment:
@kgpayne - can you confirm if this is backwards compatible? What would happen if users bump the SDK without changing this test strategy? Ideally we would not break existing implementations - either we run all new tests, or run some subset - but we definitely would not want all taps and targets to have their tests broken with their next SDK version bump.

Reply:
I decided this was easier to just test by reverting the dropped tests:

- https://github.com/MeltanoLabs/target-csv/pull/5/commits/b1dd55455f0ce804cdc08c44a88f7c13eb42e769

Reply:
✅ Legacy tests still pass! 🎉